packages/server-wallet/src/models/channel.ts

```diff
@@ -96,11 +103,15 @@ export class Channel extends Model implements RequiredColumns {
 
   static jsonAttributes = ['vars', 'participants'];
 
-  static forId(channelId: Bytes32, tx?: Transaction): Promise<Channel> {
-    return Channel.query(tx)
+  static async forId(channelId: Bytes32, tx: Transaction | undefined): Promise<Channel> {
```

Comment:
Could this method instead return an `Promise<Either<ChannelError, Channel>>`? Should it?

Reply:
Yes, and I think so!